.github/workflows/ci.yml

```diff
@@ -6,8 +6,9 @@ jobs:
   rust:
     name: deno_graph-${{ matrix.os }}
     if: |
-      github.event_name == 'push' ||
-      !startsWith(github.event.pull_request.head.label, 'denoland:')
+      (github.event_name == 'push' || !startsWith(github.event.pull_request.head.label, 'denoland:'))
+      && github.ref_name != 'deno_registry'
+      && !startsWith(github.ref, 'refs/tags/deno_')
```

Comment:
I'm going to update the webhook to use this prefix https://github.com/denoland/deno_registry2/blob/main/API.md#request